src/client/utils/hooks/useDocumentScroll.tsx

```diff
@@ -8,25 +8,26 @@ export const useDocumentScroll = (
   }) => void,
 ) => {
   const [, setScrollTop] = React.useState(0)
```

Comment:
Unrelated to this, but why isn't there a state variable here? 🙃

Reply:
Haha, I would also like to know that!